src/webgpu/gpu_test.ts

```diff
@@ -147,7 +147,12 @@ export class GPUTest extends Fixture {
     return { dst, begin: offsetDifference, end: offsetDifference + size };
   }
 
-  expectContents(src: GPUBuffer, expected: TypedArrayBufferView, srcOffset: number = 0): void {
+  expectContents(
+    src: GPUBuffer,
+    expected: TypedArrayBufferView,
+    srcOffset: number = 0,
+    generateWarningOnly: boolean = false
```

Comment:
nit: Would be nice as a named argument (in an options dictionary) so the caller always specifies what the boolean means.